BLSQ/iaso
# POLIO-943: Remove 'PV notification date'

Explain what problem this PR is resolving
- The PV notification date field should be removed from campaign base info tab
Related JIRA tickets : [POLIO-943](https://bluesquare.atlassian.net/browse/POLIO-943)

## Self proofreading checklist


## Changes
- The PV notification date has been removed
- All translations related to it have been removed
- The yup validation of it has been removed to
## How to test
- Go in campaigns
- Edit or create campaign
- Select base info tab
- There should be no "pv notification date" anymore
!!! I kept the field in the backend.
## Print screen / video

![Screenshot from 2023-04-13 17-05-58](https://user-images.githubusercontent.com/19631540/231803012-598bc18c-69e1-4dbb-8f00-7f5a338b34bb.png)


[POLIO-943]: https://bluesquare.atlassian.net/browse/POLIO-943?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
